identity/client/src/pages/users/modals/EditModal.tsx

```diff
@@ -39,34 +41,32 @@ const EditModal: FC<UseEntityModalProps<User>> = ({
       loadingDescription={t("Updating user")}
       confirmLabel={t("Update user")}
     >
+      <TextField
+        label={t("Username")}
+        value={user.username}
+        placeholder={t("Enter username or user ID")}
+        readOnly
+      />
       <TextField
         label={t("Name")}
         value={user.name}
-        placeholder={t("Name")}
+        placeholder={t("Enter name")}
         onChange={(name) => setUser({ ...user, name })}
-        errors={namedErrors?.name}
         autoFocus
       />
       <TextField
-        label={t("Username")}
-        value={user.username}
-        placeholder={t("Username")}
-        errors={namedErrors?.username}
-        disabled
-      />
-      <TextField
-        label={t("Email")}
+        label={t("Email name")}
```

Comment:
❓ I can see that this is the design spec but I'm not convinced `Email name` is the right term, I would have expected either just `Email` or `Email address`